srml/staking/src/lib.rs

```diff
@@ -901,6 +921,24 @@ decl_module! {
 		fn set_invulnerables(validators: Vec<T::AccountId>) {
 			<Invulnerables<T>>::put(validators);
 		}
+
+		/// Add reward points to validator.
+		///
+		/// At the end of the era each the total payout will be distributed among validator
+		/// relatively to their points.
```

Comment:
any disabled validators should not be paid out.

Reply:
that will happen when for example an uncle get referenced and it is an uncle from a validator from the previous era.
I will extend the structure to handle it.

Reply:
OK currently only elected validators have an exposure, if we want to reward from previous era we need to keep this exposure from previous era. Does it make sense to keep like only two era, or should we make the number of era saved configurable.

I'll go for just last era for now